README.md

```diff
@@ -35,49 +36,81 @@ npm i @libp2p/webrtc
 ## Usage
 
 ```js
-import { createLibp2pNode } from 'libp2p'
-import { webRTC } from '@libp2p/webrtc'
-import { noise } from '@chainsafe/libp2p-noise'
+import { createLibp2p } from 'libp2p'
+import { Noise } from '@chainsafe/libp2p-noise'
 import { multiaddr } from '@multiformats/multiaddr'
-import { pipe } from 'it-pipe'
-import all from 'it-all'
-
-const node = await createLibp2pNode({
-  transports: [
-    webRTC()
-  ],
-  connectionEncryption: [
-    noise()
-  ]
-})
-
-const addr = multiaddr('/ip4/0.0.0.0/udp/56093/webrtc/certhash/uEiByaEfNSLBexWBNFZy_QB1vAKEj7JAXDizRs4_SnTflsQ')
-const { stream } = await node.dialProtocol(addr, '/my-protocol/1.0.0')
-const values = await pipe(stream, all)
+import first from "it-first";
+import { pipe } from "it-pipe";
+import { fromString, toString } from "uint8arrays";
+import { webRTC } from 'js-libp2p-webrtc'
+
+const node = await createLibp2p({
+  transports: [webRTC()],
+  connectionEncryption: [() => new Noise()],
+});
+
+await node.start()
+
+const ma =  multiaddr('/ip4/0.0.0.0/udp/56093/webrtc/certhash/uEiByaEfNSLBexWBNFZy_QB1vAKEj7JAXDizRs4_SnTflsQ')
+const stream = await node.dialProtocol(ma, ['/my-protocol/1.0.0']) 
+const message = `Hello js-libp2p-webrtc\n`
+const response = await pipe([fromString(message)], stream, async (source) => await first(source))
+const responseDecoded = toString(response.slice(0, response.length))
 ```
-## API
+
+## Examples
+Examples can be found in the [examples folder](examples/README.md).
+
+## Interfaces
 
 ### Transport
 
-[![](https://raw.githubusercontent.com/libp2p/js-libp2p-interfaces/master/packages/libp2p-interfaces/src/transport/img/badge.png)](https://github.com/libp2p/js-libp2p-interfaces/tree/master/packages/libp2p-interfaces/src/transport)
+![https://github.com/libp2p/js-libp2p-interfaces/tree/master/packages/interface-transport](https://raw.githubusercontent.com/libp2p/js-libp2p-interfaces/master/packages/interface-transport/img/badge.png)
 
-`libp2p-webrtc` accepts WebRTC encapsulated addresses: `/ip4/0.0.0.0/udp/56093/webrtc/certhash/uEiByaEfNSLBexWBNFZy_QB1vAKEj7JAXDizRs4_SnTflsQ`
+Browsers can only `dial`, so `listen` is not supported.
 
-### Connection
+```js
+interface Transport {
+  [Symbol.toStringTag]: string
```

Comment:
After reading up I found out what the implementation of this line would do, but I do not get why it is useful to declare here?